syntax = "proto2";

message CvtPair {
    optional uint32 source_id = 1;
    optional uint32 convert_type = 2;
}

message CvtPairs {
    repeated CvtPair cvt_pair = 1;
}
